src/main/java/org/sunbird/workflow/service/UserBulkUploadService.java

```diff
@@ -260,27 +244,90 @@ private void processBulkUpload(HashMap<String, String> inputDataMap) throws IOEx
+                        }
+                    }
+                    if (nextRow.getCell(10) != null && nextRow.getCell(10).getCellType() != CellType.BLANK) {
+                        if (nextRow.getCell(10).getCellType() == CellType.NUMERIC) {
+                            valuesToBeUpdate.put(Constants.PINCODE, NumberToTextConverter.toText(nextRow.getCell(10).getNumericCellValue()));
+                        } else if (nextRow.getCell(10).getCellType() == CellType.STRING) {
+                            valuesToBeUpdate.put(Constants.PINCODE, nextRow.getCell(10).getStringCellValue().trim());
+                        } else {
+                            errList.add("Invalid column type. Expecting number/string format");
+                        }
+                    }
+                    if (nextRow.getCell(11) != null && nextRow.getCell(11).getCellType() != CellType.BLANK) {
+                        if (nextRow.getCell(11).getCellType() == CellType.NUMERIC) {
+                            valuesToBeUpdate.put(Constants.EXTERNAL_SYSTEM_ID, NumberToTextConverter.toText(nextRow.getCell(11).getNumericCellValue()).trim());
+                            if (!ValidationUtil.validateExternalSystemId((String)valuesToBeUpdate.get(Constants.EXTERNAL_SYSTEM_ID))) {
+                                errList.add("Invalid External System ID : External System Id can contain alphanumeric characters and have a max length of 30");
+                            }
+                        } else if (nextRow.getCell(11).getCellType() == CellType.STRING) {
+                            valuesToBeUpdate.put(Constants.EXTERNAL_SYSTEM_ID, nextRow.getCell(11).getStringCellValue().trim());
+                            if (!ValidationUtil.validateExternalSystemId((String)valuesToBeUpdate.get(Constants.EXTERNAL_SYSTEM_ID))) {
+                                errList.add("Invalid External System ID : External System Id can contain alphanumeric characters and have a max length of 30");
+                            }
+                        } else {
+                            errList.add("Invalid column type. Expecting string/number format");
+                        }
+                    }
+                    if (nextRow.getCell(12) != null && !org.apache.commons.lang.StringUtils.isBlank(nextRow.getCell(12).toString())) {
+                        if (nextRow.getCell(12).getCellType() == CellType.STRING) {
+                            valuesToBeUpdate.put(Constants.EXTERNAL_SYSTEM, nextRow.getCell(12).getStringCellValue().trim());
+                            if (!ValidationUtil.validateExternalSystem((String)valuesToBeUpdate.get(Constants.EXTERNAL_SYSTEM))) {
+                                errList.add("Invalid External System : External System can contain only alphabets and can have a max length of 255");
+                            }
+                        } else {
+                            errList.add("Invalid column type. Expecting string format");
```

Comment:
Instead of "column" -- do provide which field we are trying to parse and the expected format.